.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "submodules/mongodb-atlas-cli"]
+	path = submodules/mongodb-atlas-cli
+	url = https://github.com/mongodb/mongodb-atlas-cli
```

Comment:
```suggestion
	url = https://github.com/mongodb/mongodb-atlas-cli.git
```

Reply:
Thanks, I fixed this. For the TOC changes you mention in your comment, those are planned for a TOC cleanup ticket I plan to do once I can stage. Thank you!